templates/commandline.tmpl

```diff
@@ -63,7 +63,8 @@ kubectl config set-credentials "{{ .KubeCfgUser }}"  \
     --auth-provider-arg='client-id={{ .ClientID }}'  \
     --auth-provider-arg='client-secret={{ .ClientSecret }}' \
     --auth-provider-arg='refresh-token={{ .RefreshToken }}' \
-    --auth-provider-arg='id-token={{ .IDToken }}'
+    --auth-provider-arg='id-token={{ .IDToken }}' \
+    --auth-provider-arg='idp-certificate-authority-data={{ .IdentityProviderCA }}' \
```

Comment:
If `idp-certificate-authority-data` is empty it is [ignored](https://github.com/kubernetes/kubernetes/blob/v1.17.3/staging/src/k8s.io/client-go/plugin/pkg/client/auth/oidc/oidc.go#L133-L137).